hikey.mk

```diff
@@ -420,9 +420,33 @@ nvme-cleaner:
 ################################################################################
 # Flash
 ################################################################################
+define flash_help
+	@read -r -p "1. Connect USB OTG cable, the micro USB cable (press any key)" dummy
+	@read -r -p "2. Connect HiKey to power up (press any key)" dummy
+endef
+
+.PHONY: recovery
+recovery:
+	@echo "Enter recovery mode to flash a new bootloader"
+	@echo "Jumper 1-2: Closed (Auto power up)"
+	@echo "       3-4: Closed (Boot Select)"
+	@echo "       5-6: Open   (GPIO3-1)"
```

Comment:
@vchong +1
Like you, at first I read the text in parentheses as the meaning of the "Closed" or "Open" setting, while in fact it is the purpose of the jumper. What about adding both information?
```
Jumper 1-2: Closed (Auto power = Boot up when power is applied)
       3-4: Closed (Boot select = Recovery: program eMMC from USB OTG)
```
Regarding jumper 5-6, I think its state is irrelevant here (it is checked by UEFI, not by the boot ROM nor by l-loader.bin).